atlas/AtlasMl/tests/test_weaviate.py

```
@@ -10,161 +7,356 @@
+    # Test with empty list
+    with pytest.raises(ValueError, match="Embeddings must be a non-empty list of floats"):
+        client.add_embeddings(collection_name, [])
+    
+    # Test with non-list embeddings
+    with pytest.raises(ValueError, match="Embeddings must be a non-empty list of floats"):
+        client.add_embeddings(collection_name, "not a list")
+
+
+def test_add_embeddings_query_error():
+    """Test add_embeddings with Weaviate query error."""
+    from unittest.mock import patch
```

Ruff (0.12.2)

`import` should be at the top-level of a file

(PLC0415)